agent/src/beerocks/slave/beerocks_slave_main.cpp

```diff
@@ -166,7 +166,7 @@ static void fill_son_slave_config(beerocks::config_file::sConfigSlave &beerocks_
         beerocks_slave_conf.enable_repeater_mode[slave_num] == "1";
     son_slave_conf.hostap_iface_type = beerocks::utils::get_iface_type_from_string(
```

Comment:
No more "before this change - after this change" commits please. Use proper English and describe the change.
Suggested commit message:
> agent: slave_main: use interface name from platform db
>
> Update son slave and slave main to fetch the hostap_iface from the platform DB instead of the configuration file. 
> 
> Signed-off-by ...